src/bot/events/verification.js

```diff
@@ -1,10 +1,23 @@
 import {EventListener} from 'yuuko';
 import createLogger from 'another-logger';
 const log = createLogger({label: 'event:verification'});
+import config from '../../../config';
+import {formatDate, formatDateRelative} from '../util/discord';
 
 // Check if members are already verified when being added to a guild
 // TODO more verification hardcoding
-export default new EventListener('guildMemberAdd', async (guild, member, {db}) => {
+export default new EventListener('guildMemberAdd', async (guild, member, {db, client}) => {
+	if (member.bot) return;
+	if (guild.id !== config.TEMP_guildID) return;
+
+	await client.createMessage(config.TEMP_joiningChannelID, {
```

Comment:
I don't think this needs to be awaited - we don't rely on the resulting message and we explicitly ignore errors on it. Not awaiting will let the code underneath run while the create message request is still happening.

Reply:
Done